medusa/backup.py

```diff
@@ -385,8 +385,20 @@ def backup_snapshots(storage, manifest, node_backup, node_backup_cache, snapshot
 
         manifest_objects = list()
         if len(needs_backup) > 0:
-            manifest_objects = storage.storage_driver.upload_blobs(needs_backup, dst_path)
-
+            # If there is a plenty of files to upload it should be
```

Comment:
Could we rework the code in order to use a function that will split the `needs_backup` list into chunks of a given size, and then loop through the chunks?
It would simplify the code and make it more readable.
Something like [this](https://www.geeksforgeeks.org/break-list-chunks-size-n-python/):

```
def divide_chunks(l, n): 
    for i in range(0, len(l), n):  
        yield l[i:i + n] 

...
MAX_FILES_PER_BACKUP_CHUNK = 64
...
backup_chunks = list(divide_chunks(needs_backup, MAX_FILES_PER_BACKUP_CHUNK))
files_uploaded = 0
for batch in backup_chunks:
  logging.debug("Uploading %s files to %s.", len(batch), dst_path)
  manifest_objects += storage.storage_driver.upload_blobs(batch, dst_path)
```

What do you think?

Reply:
I got it. How do you think is it OK to move this function in medusa.storage and import when needed as auxiliary function?

Reply:
Anyway I've done it somehow, please review and let me know.